explorer/templates.go

```diff
@@ -303,5 +312,18 @@ func makeTemplateFuncMap(params *chaincfg.Params) template.FuncMap {
 		"theme": func() string {
 			return netTheme
 		},
+		"emptyArray": func(length int) []int {
```

Comment:
Another nit pick: array and slice are quite different, and this makes a slice of a certain length with zero values, so this function is better named "zeroSlice".